verifier/server/app.js

```diff
@@ -70,19 +80,8 @@ async function getQR(req, res) {
   const proofRequest = {
     id: challenge,
     circuit_id: circuitId,
-    rules: {
-      query: {
-        allowedIssuers: ['*'],
-        schema: {
-          type: schemaType,
-          url: schemaUrl,
-        },
-        req: {
-          birthDay: {
-            $lt: 20000101, // bithDay field less then 2000/01/01
-          },
-        },
-      },
+    rules: {  // FIXME: shouldn't this be an array?
+      query,
```

Comment:
Actually, it's the `query` not the `rules` that's an array in the Iden3 doc:
https://docs.iden3.io/protocol/querylanguage/#proof-request-workflow

But maybe that's just an example, not canonical. Not sure. Out of scope for this PR in any case.